tests/foreman/destructive/test_ldap_authentication.py

```diff
@@ -266,7 +266,7 @@ def test_single_sign_on_ldap_ad_server(
 
 
 def test_single_sign_on_using_rhsso(
-    module_subscribe_satellite, rhsso_setting_setup, enable_external_auth_rhsso, module_target_sat
+    enable_external_auth_rhsso, rhsso_setting_setup, module_target_sat
```

Comment:
Earlier this test required `module_subscribe_satellite`, wouldn't this test fail without CDN?

Reply:
no need for subscription these packages are available in base.